controllers/usersController.js

```diff
@@ -480,4 +481,54 @@ UsersController.getApplication = function(req, res){
         });
     });
 
+}
+
+/**
+ * Searches for users by full name or email
+ * 
+ * Req should contain 'name' which has the search term
+ */
+UsersController.search = function(req, res) {
+
+    let name = req.body.name;
+    Database.getAll(COLLECTION_NAME).then(function(result){
+        // Filter out users that don't have an application
+        let users = result.filter(user => user.application).map(user => {
```

Comment:
How can there be a user without an application?

Reply:
We didn't make any test accounts on the production server before applications went up, did we? If not, then I guess it should be fine. I can remove this.

Reply:
No, accounts came with applications. Besides, documents in the `Users` collection are created with null values so the application is technically there.

Reply:
Right. It was just something I ran into during my testing because my test database wasn't great. Removed this part.

Reply:
If you're in need of data, the cuhacking-dev firestore has all the information the production firestore had as of yesterday. You can play with that as much as you like.

None of the accounts were ported over though, so auth stuffl won't work.